docs/document/README.md

```diff
@@ -161,14 +161,20 @@ npm run build
 
 ### Testing your Build Locally
 
-Execute `serve` command to test your build locally.
+1. Execute `serve` command to test your build locally.
 
 ```sh
 npm run serve
 ```
 
 For more information about build, click [here](https://docusaurus.io/docs/2.x/deployment).
 
+2. Because versioning is used, access the link with `next` to see the documents you added or modified.<br>
```

Comment:
Doesn't the 'next' description also apply to documents started with 'npm run start'?
Shouldn't this part be noticed as well?

Reply:
No, I think we don't need to write the same thing twice.